readme.txt

```diff
@@ -0,0 +1,28 @@
+=== Legacy features for FSE ===
+Contributors: Chouby, manooweb, marianne38, sebastienserre, greglone, hugod
+Donate link: https://polylang.pro
+Tags: multilingual, bilingual, translate, translation, language, multilanguage, international, localization
+Requires at least: 5.9
+Tested up to: 6.0
+Requires PHP: 5.6
+Stable tag: 1.0
+License: GPLv3 or later
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+
+Make legacy features usable with Full Site Editor(FSE).
+
+== Description ==
+
+Currently this plugin adds the legacy widgets block in any kind of block editor especially the Full Site Editor(FSE) introduced by WordPress 5.9.
+
+By reintroducing the legacy widgets block, you can use some of the legacy widgets as Naviagation Menu for example.
+
+However with blocks theme, like Twenty Twenty-Two, you can't update menus anymore because this kind of theme doesn't support menus.
+
+This is why this plugin reintroduce the WordPress admin menus to be able to manage menus you want to use with the legacy widgets block.
+
```

Comment:
Same as above, maybe we could add the feature I mentionned.

Reply:
Done in 00b0142a40727b8586f799357a5458faa040f672